active_learning/fetch_data.py

```diff
@@ -84,6 +142,21 @@ def tf_idf_matrix(raw_documents, tokenizing_method):
 
 
 def feature_selection(tf_idf_matrix, document_classes_for_matrix, max_features, feature_names):
+    '''
+    Uses a tf idf matrix as well as the given document classes for the matrix in order to extract an array of features to use
+
+    Parameters
```

Comment:
This one and the one above will have to be filled in later, I wasn't quite sure how to add the parameter comments in as of now.